personalize.module

```diff
@@ -669,6 +671,17 @@ function personalize_library() {
       array('system', 'jquery.once'),
     ),
   );
+  $libraries['admin.campaign.list'] = array(
+    'title' => 'Personalization list',
+    'website' => '',
+    'version' => VERSION,
+    'js' => array(
+      'jQuery(document).ready(function(){jQuery(".content").tabs();});' => $campaign_list_options,
```

Comment:
I was aiming for minimum footprint, but you have a point there. Done [here](https://github.com/acquia/personalize_module/commit/bad57b3e2ea9cbebdb782ab8938f318509265a5d#diff-336c0df322b81a28385f888327c4f932R345).